packages/admin-panel/src/widgets/InputField/InputField.jsx

```diff
@@ -23,13 +23,44 @@ const getInputType = ({ options, optionsEndpoint, type }) => {
   return type;
 };
 
-export const InputField = ({ type, secondaryLabel, error, ...inputProps }) => {
+export const InputField = ({
+  type,
+  maxLength,
+  minLength,
+  secondaryLabel,
+  error,
+  ...inputProps
+}) => {
   const { options, optionsEndpoint } = inputProps;
   const inputType = getInputType({ options, optionsEndpoint, type });
   const InputComponent = InputFields[inputType];
+
+  const generateHelperText = () => {
+    if (secondaryLabel) return secondaryLabel;
+    if (maxLength && minLength !== undefined && minLength !== null) {
+      return `Must be between ${minLength} and ${maxLength} characters`;
+    }
+
+    if (maxLength) {
+      return `Max ${maxLength} characters`;
+    }
+
+    if (minLength !== undefined && minLength !== null) {
```

Comment:
Nice! I was hoping to catch you on the `if(minLength)` :stuck_out_tongue_closed_eyes: